docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -10,8 +10,9 @@ Internal authentication
 -----------------------
 
 Requests between Trino nodes are authenticated using a shared secret. For secure
-internal communication, the shared secret must be set to the same value on all
-nodes in the cluster:
+internal communication, and for any :ref:`external authentication
+<security-authentication>`, the shared secret must be set to the same value on
+all nodes in the cluster:
```

Comment:
Can you mention that this config belongs in `config.properties`?

Reply:
done